hypha.php

```diff
@@ -85,7 +85,7 @@ function index($dir = '.') {
 		if ($handle = opendir($dir)) {
 			while (false !== ($file = readdir($handle))) {
 				if ($file != "." && $file != "..") {
-					if (isAllowedFile($dir.'/'.$file)) $index[ltrim($dir.'/'.$file, "./")] = filemtime($dir.'/'.$file);
+					if (isAllowedFile($dir.'/'.$file)) $index[preg_replace("/^\.\//i","",$dir.'/'.$file)] = filemtime($dir.'/'.$file);
```

Comment:
I think this change is unrelated to the rest of the commit (IIRC it fixes installation of the .htaccess file?). Would be better to have this in a separate commit (separate PR is not needed).

Reply:
This commit could probably be set to close #92.

Reply:
This comment is still unresolved.

Reply:
done